File: resolucoes/python/lista01/ex018.py
# Faça um programa que leia o ano de nascimento de uma pessoa, calcule a idade dela e depois mostre se ela pode ou não votar.
from datetime import date

ano_atual = date.today().year

ano_nascimento = int(input('Digite o ano em que você nasceu: '))

if ano_atual - ano_nascimento >= 18:
    print('Você tem mais de/ou 18 anos, você já pode votar!')
else:
    print('Você ainda não pode votar!')

File: resolucoes/JavaScript/lista01/ex018.js
const prompt = require('prompt-sync')();

function main() {

const nascimento = Number(prompt("Qual ano você nasceu?"))
const ano = new Date().getFullYear()
const idade = ano - nascimento

if (idade >= 16) {
    console.log("Você pode votar.")
} else {
    console.log("Você não pode votar.")
}
}

module.exports = {
    main
}